### PR TITLE
Cleanup systray menu

### PR DESCRIPTION
**Pull request by TheOneRing**
Fixes: #10939

image
image

### CODE REVIEW

**Comment by TheOneRing** [comment]
When merged OWNCLOUD_MINIMAL_TRAY_MENU needs to be removed from the doc.

**Comment by michaelstingl** [comment]
> When merged OWNCLOUD_MINIMAL_TRAY_MENU needs to be removed from the doc.

PR
- https://github.com/owncloud/docs-client-desktop/pull/451

**Comment by TheOneRing** [comment]
> I see two images but I cant identify the so called minimal tray menu. I want to double check if we have used screenshots in the docs that we need to remove too. Can one help with before/after images?

Feel free to test OWNCLOUD_MINIMAL_TRAY_MENU locally.

File: changelog/unreleased/10939 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Change: Modernise systray menu
```
Comment (fmoc) [inline changelog/unreleased/10939:1]:
```suggestion
Change: Modernize systray menu
```

We usually prefer American English.

**Review by fmoc — APPROVED** [review]
Looks good to me. Approving so you can merge. Please consider my one suggestion.

**Pull request merged**